### DISCUSSION TITLE
Sonoff Z-Bridge Pro usato come router con Zigbee2MQTT

### DISCUSSION THREAD

**Discussion by lnrd00** [indomus/forum#2999] [ZigBee, Z-Wave, Thread, Matter]
Salve, avrei una domanda. 
Ho un Sonoff Z-Bridge Pro nel quale, dopo averlo flashato, ho installato un firmware di tipo "router" (`SonoffZigbeeBridgePro_router_only_cc2652.hex`) per poterlo far funzionare a distanza dall'antenna coordinator. 
Pensavo che, fornendogli una connessione wifi, esso potesse estendere la copertura della mia rete zigbee **senza collegamento diretto con l'antenna coordinator**, ma è così? Oppure antenna coordinator e router devono sempre e comunque "vedersi" ? 
Grazie in anticipo della risposta.
P.S. Questi miei dubbi ne seguono altri avuti in passato( #2934 ), dove ero riuscito a creare una rete funzionante con coordinator + router, ma lì erano a distanza ravvicinata tra loro.

Grazie in anticipo!

**Comment by KC-inDomus** [comment]
a logica se usi un firmware router, una volta installato l'unità si accende in pairing, la fai entrare in rete zigbee e hai fatto: basta tenerla alimentata e fa da repeater.

**Reply by lnrd00** [reply]
proverò a metterla nuovamente in modalità pairing per vedere se magari il problema è che non riesce ad agganciarsi alla rete wifi.
Se la sposto da dove si trova e la riconnetto alla corrente in una posizione vicino al coordinator, non ha alcun problema a entrare in funzione, ma non è lì che mi serve.

**Reply by KC-inDomus** [reply]
alla wifi non si riconnetterà, non penso sia quella la strada, se gli metti un firmware router zigbee si limita a esporre quel protocollo e fine

**Reply by lnrd00** [reply]
Mi sono anche accorto che dalla pagina web del Sonoff Zigbee Bridge Pro non posso visualizzare la mappa dei dispositivi, né permettere la connessione di nuovi dispositivi.
![IMG_6770](https://github.com/indomus/forum/assets/154708323/04490999-f200-4388-92b1-ac635de1ce44)
Inoltre, quando dalla pagina web di zigbee2mqtt faccio “permetti join” per il sonoff Zigbee Bridge Pro, ottengo il seguente errore:
![IMG_6772](https://github.com/indomus/forum/assets/154708323/b4d8a6ac-0b58-4210-b41a-84c06412341c)
Qualcuno sa di cosa potrebbe trattarsi?
Sui logs, quello che vedo è

```
Zigbee2MQTT:info  2024-03-27 10:44:55: Zigbee: allowing new devices to join via SNZBBridgePro.
Zigbee2MQTT:error 2024-03-27 10:44:55: Request 'zigbee2mqtt/bridge/request/permit_join' failed with error: 'ZdoRequest error'
Zigbee2MQTT:info  2024-03-27 10:44:55: MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{},"error":"ZdoRequest error","status":"error","transaction":"cgchy-1"}'

```
